python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -970,6 +972,11 @@ def test_show(self):
         expected = "+---+---+\n|  X|  Y|\n+---+---+\n|  1|  2|\n+---+---+\n"
         self.assertEqual(show_str, expected)
 
+    def test_stat_corr(self):
```

Comment:
what about create a df with `spark.sql(...)` and compare the results between PySpark and Connnect?

Reply:
OK